docs/administrators/logging.rst

```diff
@@ -12,69 +12,46 @@ configured regarding to logging.
-
-* **Store**: A new version of a configuration model and its resources has been pushed to the Inmanta server.
-* **Pull**: An agent pulled its resources from the Inmanta server.
-* **Deploy**: When an agent starts and ends the deployment of a certain resource.
-* **Dryrun**: Execute a dryrun for a certain resource.
-
-
-Agent logs
-----------
+Configure logging
+=================
 
-One agent produces the following three log files:
+Logging can be configured in two main ways:
 
-* ``agent-<environment-id>.log``: This is the main log file of an agent. It contains information about when the agent
-  started a deployment, which trigger caused that deployment, whether heartbeat messages are received from the server,
-  whether the agent is a primary agent, etc.
-* ``agent-<environment-id>.out``: This log file contains all the messages written to the standard output stream of the resource
-  handlers used by the agent.
-* ``agent-<environment-id>.err``: This log file contains all the messages written to the standard error stream of the resource
-  handlers used by the agent.
+- coarse grained configuration using configuration and command line options. This is sufficient in most cases.
+- fine grained configuration using a config file. Here the logging config is fully user controlled.
 
 
-Configure logging
-=================
 
-Configuration options in Inmanta config file
---------------------------------------------
+Coarse grained configuration
+----------------------------
 
 The following log-related options can be set in an Inmanta config file:
 
-* ``log-dir``
-* ``purge-resource-action-logs-interval``
-* ``resource-action-log-prefix``
+* :inmanta.config:option:`config.log-dir`: determines the folder that will contain the log files
 
-Documentation on these options can be found in the :ref:`Inmanta configuration reference<config_reference>`.
+:ref:`As command line options,<reference_commands_inmanta>` the following are available:
 
+* ``--timed-logs``: Add timestamps to logs
+* ``--log-file``: Path to the logfile, enables logging to file, disables logging to console
+* ``--log-file-level``: Log level for messages going to the logfile, options  `ERROR`, `WARNING`, `INFO`, `DEBUG`, `TRACE`
+* ``-v``: Log level for messages going to the console. Default is warnings only. -v warning, -vv info, -vvv debug and -vvvv trace.
+* ``-X``: When exiting with an error, show full stack trace.
+* ``--keep-logger-names``: When using the compiler, don't shorten logger names
 
-Change log levels server log
-----------------------------
-
-Edit the ``--log-file-level`` option in the ExecStart command of the inmanta-server service file. The inmanta-server service
-file can be found at ``/usr/lib/systemd/system/inmanta-server.service``.
+To update the server startup config when using the RPM based install, edit the inmanta-server service
+file at ``/usr/lib/systemd/system/inmanta-server.service``.
```

Comment:
This is not in-line with the standard usage of systemd. The files in `/usr/lib/systemd/system/` are files provided by the distribution package and they should not be updated by administrators. If you want to override the file, you should copy the file to `/etc/systemd/system`, which will do an override. I am also pretty sure the RPM spec file is configured to override any changes made to `/usr/lib/systemd/system/inmanta-server.service` at upgrade time. So this procedure would introduce another step in the upgrade procedure. I don't think we want that.

Reference: `man systemd.unit`